shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -336,6 +338,22 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
       // Reset for the next pass:
       flatland_layer_index++;
     }
+
```

Comment:
The old logic above needs massaging after this patch, can you just mark it with the RT tree bug, and say it should match what's done in the gfx external view embedder?

Reply:
Done